autoscaler/README.md

```diff
@@ -0,0 +1,37 @@
+# Horizontal Self Scaler container
+
+This container image watches over the number of schedulable nodes in the cluster and resizes
+the number of replicas in the required controller.
+
+Usage of pod_nanny:
```

Comment:
I think @thockin and @davidopp both agree on cluster-proportional-autoscaler, I will change all references including the name of the binary itself to this choice.

@bgrant0607 Permission to create repository named cluster-proportional-autoscaler (and delete this one) ?